src/content-tags-drawer/TagBubble.jsx

```diff
@@ -8,35 +8,45 @@ import PropTypes from 'prop-types';
 import TagOutlineIcon from './TagOutlineIcon';
 
 const TagBubble = ({
-  value, subTagsCount, implicit, level,
+  value, implicit, level, lineage, removeTagHandler,
 }) => {
   const className = `tag-bubble mb-2 ${implicit ? 'implicit' : ''}`;
   const tagIcon = () => (implicit ? <TagOutlineIcon className="implicit-tag-icon" /> : <Tag />);
+
+  const handleClick = (e) => {
+    if (e.target.value) {
+      e.target.checked = false;
+      removeTagHandler(e);
+    }
+  };
+
   return (
     <div style={{ paddingLeft: `${level * 1}rem` }}>
       <Button
```

Comment:
This should be a [`<Chip>`](https://paragon-openedx.netlify.app/components/chip/), not a `<Button>`. Chip is specifically designed to be a displayable thing with an "X" remove button on the right. We want the click event to be on the "x" only:

```javascript
      <Chip
        className={className}
        variant="light"
        iconBefore={!implicit ? Tag : TagOutlineIcon}
        iconAfter={!implicit ? Close : null}
        onIconAfterClick={handleClick}
      >
        {value}
      </Chip>
```

![Screenshot 2023-11-22 at 3 25 24 PM](https://github.com/openedx/frontend-app-course-authoring/assets/945577/018dd892-7754-4bf6-b38a-b3a775e22bd1)

Here's some updated SCSS to get you going:

```scss
.tag-bubble.pgn__chip {
  border-color: $light-300;
  border-style: solid;
  border-width: 2px;
  background-color: transparent;

  &:hover {
    .pgn__chip__icon-before * {
      color: $white;
      fill: $white;
    }
    .pgn__chip__label {
      color: $white;
    }
    background-color: $dark;
    border-color: $dark;
  }
}
```

Reply:
Oh great! I wasn't aware of `Chip` I switched it up like you suggested and it works quite well, thanks!